metrics/write.go

```diff
@@ -341,9 +341,8 @@ func (c *Client) Store(ctx context.Context, req *prompb.WriteRequest) error {
 			line = scanner.Text()
 		}
 		err = fmt.Errorf("server returned HTTP status %s: %s", httpResp.Status, line)
+		log.Println(err)
 	}
-	if httpResp.StatusCode/100 == 5 {
-		return err
-	}
```

Comment:
Also removed this block as it seems like the error would be assigned in the previous if block anyway on 5xx? Can add back in if I'm wrong somewhere